crates/release_plz_core/src/release.rs

```diff
@@ -131,9 +136,24 @@ fn release_package(
         info!("published {}", package.name);
     }
 
+    if input.gh_release {
+        publish_release(package);
+    }
+
     Ok(())
 }
 
+async fn publish_release(package: &Package) {
+    let page = octocrab::Octocrab::default()
+        .repos(package.authors.join(", "), package.clone().name)
```

Comment:
This should be the info contained in the GitHub url of the repository.
For example, for this repository it's `.repos("MarcoIeni", "release-plz")`.
This info doesn't always correspond to the cargo.toml file.